python/py-hdbscan/Portfile

```diff
@@ -69,7 +69,9 @@ if {${name} ne ${subport}} {
                         "LDFLAGS=${configure.ld_archflags} -L${prefix}/lib/libomp -lomp" \
                         DYLD_LIBRARY_PATH=${prefix}/lib/libomp
 
-    test.run            yes
+    if {${python.pep517}} {
+        build.cmd-append    --skip-dependency-check
```

Comment:
this doesn't seem like a good idea/proper fix; we do want to make sure the dependencies are checked. What issue are you trying to work around?

Reply:
The `setup.py` looks for them and throws an error even though they're installed. This flag fixes that issue.

Reply:
that can't be right, if installed it should be found. Please spend some time to figure out the root cause of this.

Reply:
Sorry, I don’t have the `setup.py` expertise necessary to track this down.

Reply:
the issue has nothing to do with the `setup.py`, since it's apparently only an issue with the `pep517` builds it will be in the `pyproject.toml` file. I presume the issue you were seeing was (that is as described [here](https://trac.macports.org/ticket/67136)):
```
ERROR Missing dependencies:
	oldest-supported-numpy
```

so patching the `pyproject.toml` file to have that as `numpy` is what should be done.

Reply:
Okay thanks. Patch added.